src/gui/static/src/app/components/layout/hardware-wallet/hw-added-dialog/hw-added-dialog.component.ts

```diff
@@ -58,4 +80,10 @@ export class HwAddedDialogComponent extends HwDialogBaseComponent<HwAddedDialogC
     this.walletService.saveHardwareWallets();
     this.closeModal();
   }
+
+  openHelp() {
+    this.dialog.open(HwPassphraseHelpDialogComponent, <MatDialogConfig> {
+      width: '450px',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
      width: "450px",
```